docs/testplan/GCU-Dynamic-ACL-testplan.md

```diff
@@ -0,0 +1,309 @@
+# Dynamic ACL Update via GCU Test Plan
+
+**Table of Contents**
+
+
+## Overview
+
+This test plan will certify that Generic Config Updater (GCU) is able to properly add, remove, and update ACL Table Types, ACL Tables, and ACL Rules.
+
+## Testbed
+
+The test will run on T0 testbeds.
+
+## Setup Configuration
+
+No setup pre-configuration is required, the test will configure and return the testbed to its original state.
+
+## Testing Plan
+
+To test the capability of GCU to dynamically update ACLs, we will:
```

Comment:
1. Please list all the test cases in the test plan. You can take any test plan in https://github.com/sonic-net/sonic-mgmt/tree/master/docs/testplan as a reference.
2. For each single test case, please clarify the test setup, test steps and expectations

Reply:
For example,

- Test case 1 - Quarantine a port with GCU
- Test case 2 - Un-quarantine a port with GCU
- Test case 3 - xxxx

The target is to cover all the scenarios that are going to be used in production.